test-suite/coq-makefile/plugin1/run.sh

```diff
@@ -23,4 +27,5 @@ sort > desired <<EOT
 ./test/test.v
 ./test/test.vo
 EOT
+(coqc -config | grep -q "NATIVE_COMPILER_DEFAULT=yes") || sed -i '/\.coq-native/d' desired
```

Comment:
It seems that this `sed` invocation is not compatible with the macOS `sed` (this is what makes the macOS test fail).

Reply:
do you think `sed -i -e '/\.coq-native/d' desired` would be OK?

or `sed -i -e 's/^.*\.coq-native.*$//' desired`?

(assuming each file is in its own line… but untested)

Reply:
Why not do
~~~bash 
cat > desired.in <<EOT
always used files
EOT
if native configured; then
cat >> desired.in <<EOT 
native stuff
EOT
sort desired.in > desired
~~~
?
instead of generating the file with everything then deleting stuff

Reply:
OSX sed hates `-i` with no backup file, so if you want to use sed you need `sed -i.bla ...`

Reply:
Indeed, fixed, thanks!

Reply:
Oh interesting. Mac sed also takes `sed -i ‘’`, but that fails on GNU sed (which wants `-i`), so @SkySkimmer’s fix is better.